.github/workflows/ci.yml

```diff
@@ -18,6 +18,7 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: 18
+          cache: 'npm'
```

Comment:
This make sure when installing `npm` packages cache is used, since we need to set up `npm` multiple times - `actions/setup-node@v3` will install `node` and install packages, if I recall correctly.
LF test does not work like this now (there's no package to install when setting up `node`, it's handled by the LF executable); when #157 is merged we can use cache.
Hopefully I made thing clear......